File: CPP/Leetcode/7-reverse-integer/7-reverse-integer.cpp
class Solution {
public:
    int reverse(int x) {
        int cnt=0;
        while(x!=0)
        {
            int d=x%10;
            if(cnt>INT_MAX/10 || cnt<INT_MIN/10) return 0;
            cnt=cnt*10 + d;
            x=x/10;
        }
        return cnt;
    }
};

File: Some Leetcode and GFG Solutions/7-reverse-integer/7-reverse-integer.py
class Solution(object):
    def reverse(self, x):
        """
        :type x: int
        :rtype: int
        """
        sign=1 if x>0 else -1
        x=sign*(int(str(abs(x))[::-1]))
        return x if pow(-2,31)<= x<=(pow(2,31)-1) else 0